app/autoUpdate.ts

```diff
@@ -16,6 +17,11 @@ const clearNotification = ( payload: { id: string } ) =>
 
 // Thow error incase there is an issue in updating
 autoUpdater.on( 'error', ( error ) => {
+    if ( process.platform === 'linux' || process.platform === 'win32' ) {
+        updateSNAPP( [`--update-failed`, `--appId:safe.browser`] );
```

Comment:
can we pass the error message across here too? Or where would this info be shown?

Reply:
In the browser we throw a dialog box. Displaying the error, even though the window is still hidden. On Safe-network-apps side Im not sure if we should show the error there, but rather only say app-update-failed?

Reply:
>In the browser we throw a dialog box. Displaying the error, even though the window is still hidden.
That sounds like terrible UX. An error from something I don't know is running...

We should for sure show some error indicative of the failure. But will we want to know the actual reason? Maybe not in the UI, but maybe for logging eg? If it's possible to pass the error message across, perhaps that's useful